benchmark_runner/workloads/workloads_operations.py

```diff
@@ -359,7 +359,7 @@ def __get_metadata(self, kind: str = None, status: str = None, result: dict = No
         if 'bootstorm' in self._workload:
             metadata.update({'vm_os_version': 'fedora37'})
         if 'windows' in self._workload:
-            metadata.update({'vm_os_version': 'win_10'})
+            metadata.update({'vm_os_version': 'windows_server_2019'})
```

Comment:
Where is this used?  I don't see this in any of the golden files.

Reply:
Its not in yaml, its upload to ElasticSearch

Reply:
Confirmed that this is only for Elasticsearch.